### ISSUE TITLE
[feature] statistics aggregation

### ISSUE THREAD

**Issue by tetraf** [modess/git-pretty-stats#34]
It may be interesting to have a main dashboard with statistics from all repositories. Contributors overall activities, projects stats, etc.

**Comment by modess** [comment]
I agree, the issue is that crunching commits is a quite heavy operation (especially on large repositories). With a good caching mechanic and perhaps synchronous parsing of repositories it could be achieved.

**Comment by tetraf** [comment]
Yes. A possible solution could be to use git hooks to trigger statistics update. But it would be quite intrusive.
